src/NHSD.BuyingCatalogue.Database/Tables/CataloguePrice.sql

```diff
@@ -0,0 +1,17 @@
+﻿CREATE TABLE dbo.CataloguePrice
+(
+    CataloguePriceId INT IDENTITY(1,1) NOT NULL,
+    CatalogueItemId NVARCHAR(14) NOT NULL,
```

Comment:
Worth noting all of the other tables in BC have the types lower case e.g. varchar. Also nvarchar isnt widley used in BC (instead varchar is used). Do we want to keep it the same for consistency?

Reply:
OK will change them.